DependencyInjection/Compiler/AddDependencyCallsCompilerPass.php

```diff
@@ -317,13 +317,20 @@ public function applyDefaults(ContainerBuilder $container, $serviceId, array $at
 
         $definition->addMethodCall('setLabel', array($label));
 
+        $persistFilters = $container->getParameter('sonata.admin.configuration.filters.persist');
+        // override default configuration with admin config if set
         if (isset($attributes['persist_filters'])) {
-            $persistFilters = (bool) $attributes['persist_filters'];
-        } else {
-            $persistFilters = (bool) $container->getParameter('sonata.admin.configuration.filters.persist');
+            $persistFilters = $attributes['persist_filters'];
+        }
+        $filtersPersister = $container->getParameter('sonata.admin.configuration.filters.persister');
+        // override default configuration with admin config if set
+        if (isset($attributes['filters_persister'])) {
+            $filtersPersister = $attributes['filters_persister'];
+        }
```

Comment:
Don't you want to do this in a dedicated pull request, for ALL the settings that share that behavior ?